docs/meetings/web/2021-12-29.md

```diff
@@ -0,0 +1,10 @@
+# Web meeting 29 Dec 2021
+
+**Present:** Aapeli, Darren, Lucas
+
+- Updates on next js - nearly done
+  - Review
+  - Rebase
+  - Docker container
+- Storybook don't need whole pages
+- Talking about how to language changing implementation
```

Comment:
How to language changin implementation?

Reply:
Something about how to handle routes like `/en/community/...` if someone who has their device/browser set to prefer `fr` (French). I think we said unless the user explicitly override with a language option thing we'll add in future, always redirect to their preferred language?

Reply:
He means my grammar is bad